pjsip4net/DefaultSipUserAgent.cs

```diff
@@ -75,7 +75,7 @@ public ISipUserAgent Start()
             Container.RegisterAsSingleton(localRegistry.SipTransport);
 
             var mediaApiProvider = Container.Get<IMediaApiProvider>();
-            mediaApiProvider.CreateMediaTransport(localRegistry.RtpTransport.Config);
+            mediaApiProvider.CreateMediaTransport(localRegistry.SipTransport.Config);

```

Comment:
I think we should rather reuse bound and public addresses from SIP transport creating a RTP transport instead (https://github.com/siniypin/pjsip4net/blob/5c2b567b4ca64c98e697586b672ad8a476e6d87d/pjsip4net/DefaultSipUserAgent.cs#L71). pjsip does not support neither TCP not TLS as RTP transports.